test/parallel/test-event-target.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const props = ['NONE', 'CAPTURING_PHASE', 'AT_TARGET', 'BUBBLING_PHASE'];
+
+for (const prop of props) {
+  const desc = Object.getOwnPropertyDescriptor(Event, prop);
+  assert.strictEqual(desc.writable, false);
+  assert.strictEqual(desc.configurable, false);
+  assert.strictEqual(desc.enumerable, true);
+}
```

Comment:
can we check the values are 0,1,2 and 3 for each event phase? 

`const props = [{'NONE', 0} ...]`

then use `ObjectKeys`

Reply:
Done, please check.

Reply:
The original change wasn't needed - this should already be checked when the WPTs are run. Sorry for the extra work :(

Reply:
It is fine, I have changed it either ways :)